internal/release/services/release/release_test.go

```diff
@@ -66,3 +73,276 @@ func Test_RegisterReleaseWithServer(t *testing.T) {
 		}
 	})
 }
+
+// Test_PublishRelease checks:
+// - if correct response is returned when release is published
+// - if error is returned when PutItem fails
+func Test_PublishRelease(t *testing.T) {
+	t.Parallel()
+
+	t.Run("when new release is published", func(t *testing.T) {
+		db := &mocks.DynamoDB{}
+
+		svc := &ReleaseService{Db: db}
+
+		req := &release.PublishRequest{
+			Type:         "test type",
+			Organization: "test org",
+			Name:         "test",
+			Version:      "v1.0.0",
+			Description:  "test desc",
+			Links: []*release.Link{
+				{
+					Title: "test title",
+					Url:   "http://www.google.com",
+				},
+			},
+		}
+
+		res, err := svc.Publish(context.TODO(), req)
+
+		if err != nil {
```

Comment:
(for info) We can simplify our unit tests by using the "require" package, see https://pkg.go.dev/github.com/stretchr/testify/require#NoError for an example.